metadeploy/api/serializers.py

```diff
@@ -602,3 +612,23 @@ class Meta:
             "company_logo",
             "favicon",
         )
+
+
+class ScratchOrgJobSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ScratchOrgJob
+        fields = (
+            "id",
+            "plan",
+            "email",
+            "enqueued_at",
+            "job_id",
+            "created_at",
+            "edited_at",
+            "status",
+            "canceled_at",
+            "config",
```

Comment:
There's data in the scratch org config dict which is considered secret (such as access_token). Can we serialize just the keys that are needed by the frontend (if any) rather than the entire config?

Reply:
Yeah; I don't know what the frontend might need at all, so I'll default to omitting it for now, and add in what we need as we need it.